modules/manage/pages/kubernetes/k-whole-cluster-restore.adoc

```diff
@@ -0,0 +1,6 @@
+= Whole Cluster Restore for Disaster Recovery in Kubernetes
```

Comment:
Curious to know why docs is using "Whole" here? Isn't "Cluster Restore for Disaster Recovery..." equivalent? Do we have partial cluster restore? I could not find that...Seems like "Whole" here is superfluous. Is this simply a carryover from internal eng speak?

Reply:
Why not just call this Disaster Recovery?

Reply:
Kat has plans for a DR guide: https://github.com/redpanda-data/documentation-private/issues/2112